student.rb

```diff
@@ -3,8 +3,7 @@
 
 class Student < Person
   include HighFivable
-  attr_reader :age, :phase
-  attr_accessor :name
+  attr_reader :phase
```

Comment:
Phase is still in both Student an Educator -- you can pull up the attr_reader even without pulling up the initialization.